packages/web/src/components/Space/usePictureInPicture.ts

```diff
@@ -0,0 +1,122 @@
+import { useEffect, useRef } from "react";
+
+import { RemotePeer } from "./RemotePeer";
+
+interface UsePictureInPictureOptions {
+  remotePeers: RemotePeer[];
+}
+
+const pipVideo = document.createElement("video");
+const renderer = document.createElement("canvas");
+const ctx = renderer.getContext("2d");
+
+// @ts-ignore
```

Comment:
These lines are used for debugging. You may use :

- `document.body.appendChild(PIP_CANVAS)` to see the actual canvas
- `PIP_VIDEO.requestPictureInPicture()` to trigger the PIP yourself